### DISCUSSION TITLE
Some tags can not be seen in search

### DISCUSSION THREAD

**Discussion by Fred-Vatin** [tadashi-aikawa/obsidian-another-quick-switcher#237] [Q&A] [answered]
Obsidian| OS                           | plugin
------- | ---------------------------- | -------
1.5.12  | Win 10 x64 pro (french) 22H2 | v11.4.0

Hi,

I have this note with several tags.
![image](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/22337329/f9da9c7e-2f33-4687-ac55-a80a385e4405)

Then I search for them but those tags can not be seen even after restarting obsidian.
![2024-05-10_10-20-30](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/22337329/d68739f5-9a8e-4e16-97f8-d9ee8d5b6418)

No problem with _better command palette_. The tags are on top.
![image](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/22337329/757076bf-0bf0-4017-99d8-9ca3d75c6e19)

My _landmark search_ is as is. I have less than 5 files on my vault currently.
![image](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/22337329/e676d521-80d2-4e0f-b1d6-585d5c6992a2)

Any idea what happens ?

**Comment by Fred-Vatin** [comment, marked as answer]
It seems the problem was the file was open and active. Good to know.

But, I have another problem. Even if this option 👇🏻 is ON, `enter` will open the note in the current tab instead of jumping to the tab where it is already open. It works as intended with `ctrl+enter` but I wonder why it would not just work with `enter`.
![image](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/22337329/dcf7d577-5a71-4d8a-a16c-a47df493bbc8)

**Reply by tadashi-aikawa** [reply]
@Fred-Vatin 

> enter will open the note in the current tab instead of jumping to the tab where it is already open. It 

It is as expected.

This feature was implemented in response to a feature request from https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/issues/216#issuecomment-1874033045. As I do not require this feature myself, I have asked the original requester to verify the specifications."

**Comment by Fred-Vatin** [comment]
Thanks. No matter since I can change the shortkeys 👍🏻
